### DISCUSSION TITLE
Multiline content in a tab?

### DISCUSSION THREAD

**Discussion by thephez** [executablebooks/meta#948] [Q&A]
Hi, I'm trying insert multiple lines into a tab. When I build the docs, the text is all merged onto a single line. Just wondering: 1. is it possible to have multiple lines on a tab, and 2. if so, what syntax am I missing?

Here's an example of what I'm attempting:

``` rst
.. tab-set::

    .. tab-item:: Mainnet

      ufw allow ssh/tcp
      ufw limit ssh/tcp
      ufw allow 9999/tcp
      ufw logging on
      ufw enable
```

And the output shows up as:
```ufw allow ssh/tcpn ufw limit ssh/tcp ufw allow 9999/tcp ufw logging on ufw enable```

**Comment by chrisjsewell** [comment]
The content is just regular restructuredtext, so you been to do the same as if it wasn't in a tabset, e.g. using code-block. See also: https://sphinx-design.readthedocs.io/en/furo-theme/tabs.html#tabbed-code-examples

**Reply by thephez** [reply]
Hmm, okay. Thanks for the quick response. I initially had it in a code-block (working okay), but took it out because I couldn't figure out how to change the tab name to something other than the programming language. I guess I will play around with it some more.
